src/features/apiDelegation/offered/ChooseApiPage/ApiSearchResult.tsx

```diff
@@ -13,9 +13,10 @@ import { ErrorPanel } from '@/components';
 import { ApiActionBar } from '../../components/ApiActionBar';
 
 import classes from './ChooseApiPage.module.css';
+import type { CustomError } from '@/dataObjects';
 
 interface ApiSearchResultsProps {
-  error?: FetchBaseQueryError | SerializedError;
+  error?: FetchBaseQueryError | SerializedError | CustomError;
```

Comment:
Neither of the types `FetchBaseQueryError` or `SerializedError` has the `statusCode` property. Added a third type which has both `message` and `statusCode` properties.